### PR TITLE
ClouDNS. Automated testing

### PR DESCRIPTION
**Pull request by pragmaton**
ClouDNS were kind enough to provide us with test account for CI. But I'm not sure how to provide it to you.... Who may I contact on this topic?

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -85,6 +86,11 @@ jobs:
       CLOUDFLAREAPI_KEY: ${{ secrets.CLOUDFLAREAPI_KEY }}
       CLOUDFLAREAPI_TOKEN: ${{ secrets.CLOUDFLAREAPI_TOKEN }}
       CLOUDFLAREAPI_USER: ${{ secrets.CLOUDFLAREAPI_USER }}
+#
+      CLOUDNS_AUTH_ID: ${{ secrets.CLOUDNS_AUTH_ID }}
+      CLOUDNS_AUTH_PASSWORD: ${{ secrets.CLOUDNS_AUTH_PASSWORD }}
+      CLOUDNS_SUB_AUTH_ID: ${{ secrets.CLOUDNS_SUB_AUTH_ID }}
+      CLOUDNS_DOMAIN: ${{ secrets.CLOUDNS_DOMAIN }}
```
Comment (tlimoncelli) [inline .github/workflows/build.yml:93]: Please move CLOUDNS_DOMAIN above CLOUDNS_AUTH_ID

**Comment by tlimoncelli** [comment]
Great to hear they've supplied a test account.  You can use https://transfer.secretoverflow.com/u/tlimoncelli to securely send me credentials.

File: .github/workflows/build.yml (hunk 1 above)
Comment (pragmaton) [reply .github/workflows/build.yml]: Done. plz check

**Comment by pragmaton** [comment]
> Great to hear they've supplied a test account. You can use https://transfer.secretoverflow.com/u/tlimoncelli to securely send me credentials.

Thanks. Sent, plz check

**Comment by tlimoncelli** [comment]
Looks like it worked! https://github.com/StackExchange/dnscontrol/runs/2184116963?check_suite_focus=true

**Pull request merged**